### PR TITLE
pin v0.1.4 webhook

### PR DESCRIPTION
**Pull request by luthermonson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ bin
 *.DS_Store
 .idea
 *.iml
+.charts-build-scripts
```
Comment (PennyScissors) [inline .gitignore:5]: nit: can you revert this?

File: packages/rancher-webhook/package.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.1.4-rc1/rancher-webhook-0.1.4-rc1.tgz
-packageVersion: 0
```
Comment (PennyScissors) [inline packages/rancher-webhook/package.yaml:2]: Why was this removed?

Comment (luthermonson) [reply packages/rancher-webhook/package.yaml:2]: accident i would suspect. putting it back

**Review by PennyScissors — APPROVED** [review]
